app/services/fact_parser.rb

```diff
@@ -179,10 +179,4 @@ def normalize_interfaces(interfaces)
   def not_implemented_error(method)
     "#{method} fact parsing not implemented in #{self.class}"
   end
-
-  def is_numeric?(string)
```

Comment:
Didn't see any instance of this in the salt/chef/ansible parsers so I guess it's OK to remove w/o deprecation